monitoring/src/check-page/common-functions.ts

```diff
@@ -203,23 +311,25 @@ export const checkPrivacySettingsPanelIsOpen = async (
 export const clickSaveAndCloseSecondLayer = async (
 	config: Config,
 	page: Page,
-	isAmp: boolean = true
+	isAmp: boolean = false,
 ) => {
 	log_info(`Clicking on save and close button: Start`);
 	let saveAndExitButton;
-	if(isAmp) {
+	if (isAmp) {
 		saveAndExitButton = page
 			.frameLocator(ELEMENT_ID.AMP_CMP_CONTAINER)
-			.frameLocator(ELEMENT_ID.CMP_CONTAINER).last()
+			.frameLocator(ELEMENT_ID.CMP_CONTAINER)
+			.last()
 			.locator(ELEMENT_ID.TCFV2_SECOND_LAYER_SAVE_AND_EXIT);
 	} else {
 		saveAndExitButton = page
 			.frameLocator('[src*="' + config.iframeDomainSecondLayer + '"]')
 			.locator(ELEMENT_ID.TCFV2_SECOND_LAYER_SAVE_AND_EXIT);
 	}
 
-	await saveAndExitButton.click();
+	console.log('saveAndExitButton', saveAndExitButton);
```

Comment:
console.log still needed here or should it be log_info?

Reply:
Just removed this.